app/ante/ante_test.go

```diff
@@ -42,14 +42,16 @@ func (suite *AnteTestSuite) TestValidEthTx() {
 	addr2, _ := newTestAddrKey()
 
 	acc1 := suite.app.AccountKeeper.NewAccountWithAddress(suite.ctx, addr1)
+	_ = acc1.SetCoins(newTestCoins())
 	suite.app.AccountKeeper.SetAccount(suite.ctx, acc1)
-	err := suite.app.BankKeeper.SetBalances(suite.ctx, acc1.GetAddress(), newTestCoins())
-	suite.Require().NoError(err)
+	// err := suite.app.BankKeeper.SetBalances(suite.ctx, acc1.GetAddress(), newTestCoins())
+	// suite.Require().NoError(err)
```

Comment:
Is there a reason why all these commented lines are still in here? These should get removed.

Reply:
I left all the commented lines for ease of upgrading later on. I can remove it and reference the commit on an issue if that's better

Reply:
No thats fine. I was just curious

Reply:
I think you are right and I should clean up the comments, it doesn't look clean